packages/device_info/device_info/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## 2.0.4
```

Comment:
`## NEXT`

We shouldn't be pushing updates to deprecated plugins unless it's critical.